### PR TITLE
feat: move booking between vehicles

### PR DESCRIPTION
**Pull request by Graborg**

### CODE REVIEW

**Comment by Graborg** [comment]
[Support admin to move booking from one driver to another](https://trello.com/c/1OuJZJ7R/332-support-admin-to-move-booking-from-one-driver-to-another)

**Comment by izabellalarsson** [comment]
I seem to be able to move one booking, but when I try with one more it resets 🤔

**Comment by izabellalarsson** [comment]
Might be a glitch cuz now it worked.. Nice work.

File: packages/route-optimization-jsprit/src/main/java/com/predictivemovement/route/optimization/VRPShipments.java [hunk 1]
```diff
@@ -17,6 +17,8 @@
     private static final int SHIPMENT_DEFAULT_VOLUME = 19 * 18 * 14;
     private static final int SHIPMENT_DEFAULT_WEIGHT = 1;
 
+    private static final String REQUIRES_TEANSPORT_ID = "requires_transport_id";
```
Comment (momentiris) [inline packages/route-optimization-jsprit/src/main/java/com/predictivemovement/route/optimization/VRPShipments.java:20]:
```suggestion
    private static final String REQUIRES_TRANSPORT_ID = "requires_transport_id";
```

Comment (Graborg) [reply packages/route-optimization-jsprit/src/main/java/com/predictivemovement/route/optimization/VRPShipments.java:20]: ❤️

**Review by raduachim — APPROVED** [review]
works really nice

**Pull request merged**